HelpSource/Guides/standalones.schelp

```diff
@@ -1,212 +1,356 @@
+(
+// define the name for the new app and its location:
+~newAppName = "StehAllein";
+~newAppLocation = "~/Desktop".standardizePath;
+
+// --- nothing to be changed below this line --- \\
+// some helper functions
+~readText = { |path|
+	var f = File(path, "r"); var t = f.readAllString; f.close; t;
+};
+~writeText = { |path, string| File(path, "w").write(string).close; };
+~copyFolder = { |orig, dest|
+	var str = "cp -r" + quote(orig) + quote(dest);
+	str.postcs;
+	unixCmd(str);
+};
+
+// make some needed paths and folders
+~pathToThisApp = String.scDir.dirname.dirname;
+~thisAppName = ~pathToThisApp.basename.splitext.first;
+~pathToNewApp = ~newAppLocation +/+ ~newAppName ++ ".app";
+~newAppResDir = ~pathToNewApp +/+ "Contents/Resources";
+~newAppSupportDir = Platform.userConfigDir.dirname +/+ ~newAppName;
+);
+
+
+(
+// 1. make a copy of thisApp with the newAppName:
+// can take a while on slow harddisks.
+// wait for RESULT = 0 to show!
+
+if (File.exists(~pathToNewApp).not) {
+	~copyFolder.(~pathToThisApp, ~pathToNewApp);
+	File.mkdir(~newAppSupportDir);
+} { warn("supportDir % already exists.".format(~pathToNewApp)) };
+);
+
+(
+// 2. fixups in the new app:
+// a. fix the Info.plist file by replacing the name:
+~infoPlistPath = ~pathToNewApp +/+ "Contents/Info.plist";
+// get its string, replace the SC names, write again
+~infoString = ~readText.(~infoPlistPath);
+~executableIndex = ~infoString.find("<key>CFBundleExecutable</key>");
+~nameToReplace = ~infoString.copyRange(
+	~infoString.find("<string>", offset: ~executableIndex) + 8,
+	~infoString.find("</string>", offset: ~executableIndex) - 1
+);
+
+"replacing bundleName % in plist file at these locations: "
+.postf(~nameToReplace.cs);
+~foundNameLocs = ~infoString.findAll(~nameToReplace).postln;
+
+~newInfoString = ~infoString.replace(~nameToReplace, ~newAppName );
+~writeText.(~infoPlistPath, ~newInfoString);
+// ~readText.(~infoPlistPath);
+
+// b. rename the binary file:
+"renaming macos binary to: ".post;
+// fixups in the new app - 2. rename the binary inside the app folder
+unixCmd("mv"
+	+ quote(~pathToNewApp +/+ "Contents/MacOS/" +/+ ~nameToReplace)
+	+ quote(~pathToNewApp +/+ "Contents/MacOS/" ++ ~newAppName).postln
+);
+);
+
+
+// 3. *** time for the first wakeup kiss *** //
+unixCmd("open" + ~pathToNewApp);
+
+// ... and independence test:
+// IN THE NEW APP, run this code:
+Platform.userAppSupportDir.postcs;
+::
 
-	Your version of Array.sc will now overide the default.
+NOTE:: THIS SHOULD END WITH ~newAppName! If it has the original ~thisAppName (the one you started from, e.g. ~/././SuperCollider), the new app will not be interfere with SuperCollider! So, delete it again, and start over using a recommended version of SC3 (see tested list on top)!
 
-	## teletype::Standalone Resources/SCcube.icns:: can be overridden to provide a custom icon.
+code::
+/*
+// if it failed, delete the app
+unixCmd("rm -r" + quote(~pathToNewApp));
```

Comment:
or `rm -ir` like you've suggested above—it'll check on each directory to be deleted rather than on each file